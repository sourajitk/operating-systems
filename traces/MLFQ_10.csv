1,3,5
2,5,7
3,7,9
4,9,11
5,11,13
6,13,15
7,15,17
8,17,19
9,19,21
10,21,23
11,23,25
12,25,27
13,27,29
14,29,31
15,31,33
16,33,35
17,35,37
18,37,39
19,39,41
20,41,43
