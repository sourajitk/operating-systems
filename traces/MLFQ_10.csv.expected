1,84
2,123
3,160
4,195
5,228
6,259
7,288
8,315
9,340
10,363
11,384
12,403
13,420
14,435
15,448
16,459
17,468
18,475
19,480
20,483
